specification/resources/resource-manager/Microsoft.Resources/stable/2019-05-01/resources.json

```diff
@@ -59,6 +59,321 @@
         }
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{groupId}/providers/Microsoft.Resources/deployments/{deploymentName}": {
```

Comment:
@Tiano2017 let's make the parameter token consistent across all operations
let's make it managementGroupId as mentioned on line 215

Reply:
sure. will fix it.